README.md

```diff
@@ -26,6 +26,25 @@ On Windows, there are binary wheel you can install without MySQLConnector/C or M
 `sudo yum install python3-devel `  # Red Hat / CentOS
 
 `brew install mysql-connector-c`  # macOS (Homebrew)
+#### MySQL note
```

Comment:
* Insert blank line before heading
* Note for MySQL on macOS, because this issue is macOS specific.